packages/sdks-tests/src/e2e-tests/custom-breakpoints.spec.ts

```diff
@@ -273,23 +264,6 @@ breakpoints: {
       const column2 = page.locator(`text=Column 2`);
 
       await expect(column2).toHaveCSS('color', 'rgb(126, 211, 33)');
-
-      // Skipping this image test for react-native.
-      // Its difficult to locate the image in react-native as css selectors don't work as expected.
-      if (!checkIsRN) {
-        const image = page.locator(`.builder-block:has(img.builder-image)`);
-
-        const expectedImageCss: Record<string, string> = {
-          display: 'flex',
-          width: '156px',
-          'max-width': '250px',
-        };
-
-        await expectStylesForElement({
-          locator: image,
-          expected: expectedImageCss,
-        });
```

Comment:
why did you remove this test?

Reply:
It was not being executed before due to the condition `if(!checkRn)` where checkRn was a function. I have corrected the test case now.